tests/e2e/lib/backup.go

```diff
@@ -24,6 +25,9 @@ func CreateBackupForNamespaces(ocClient client.Client, veleroNamespace, backupNa
 			SnapshotMoveData:         &snapshotMoveData,
 		},
 	}
+	if snapshotVolumes {
+		backup.Spec.SnapshotVolumes = pointer.Bool(true)
```

Comment:
maintainers, I never used this flag. This is needed if the PV is not in the same namespace as one of in `IncludedNamespaces`?

Reply:
PVs are cluster-scoped and are not in any namespace. PVCs that are not in an included namespace will be excluded from backup.
This flag tells Velero to snapshot PVCs that are 1) included in the backup, and 2) not backed up using fs-backup. If this is false, then non-fs-backup PVCs will not be snapshotted.